Directory.Build.props

```diff
@@ -257,7 +257,7 @@
     <LangVersion>latest</LangVersion>
     <LangVersion Condition="'$(Language)' == 'C#'">preview</LangVersion>
     <!-- Enables Strict mode for Roslyn compiler -->
```

Comment:
nit: update comment?

Reply:
I think we can delete it. It is kind of redundant 😄